pom.xml

```diff
@@ -119,7 +119,8 @@
 		<module>wicket-native-websocket</module>
 		<module>wicket-bean-validation</module>
 		<module>wicket-user-guide</module>
-	</modules>
+        <module>wicket-migration</module>
```

Comment:
I think it would be good to rename this module (and its Java package) to include `9_to_10`. This way it could stay for several major releases and it won't collide with the next migration, e.g. `10_to_11`

Reply:
Or even better let's introduce an intermediate Maven module for `migration` and then all `x_to_y` modules will be children of it.

Reply:
You are right! 
The user can use `-DactiveRecipes=org.apache.wicket.MigrateToWicket10` instead of `-DactiveRecipes=org.apache.wicket.BestPractices` if (s)he wants to execute only the steps for a specific version.

Reply:
Indeed! Hadn't documented that explicitly above, but that's how one can selectively update to an intermediate version if so desired. Would you then still want to introduce any naming or package changes before we resolve this conversation thread?

Reply:
No need of renaming!